iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -90,6 +93,7 @@ private async Task InitializeAndOpenClientAsync()
                         // If the device client instance has been previously initialized, then dispose it.
                         if (s_deviceClient != null)
                         {
+                            await s_deviceClient.CloseAsync();
```

Comment:
Happen to notice, some places you have used a token and not others? 

await s_deviceClient.CloseAsync(cts.Token);

Reply:
Some samples in this repo have a cancellation token for the sample, and others do not. When the sample already had a cancellation token, I passed it into the new call to deviceClient.CloseAsync accordingly

Reply:
Gotcha. For this one I remember seeing it though.